bench-tps/src/cli.rs

```diff
@@ -254,6 +256,13 @@ pub fn build_args<'a, 'b>(version: &'b str) -> App<'a, 'b> {
                 .takes_value(false)
                 .help("Submit transactions with a TpuClient")
         )
+        .arg(
+            Arg::with_name("tpu_use_quic")
+                .long("tpu-use-quic")
+                .takes_value(false)
```

Comment:
The similar parameter in `validator/src/main.rs` doesn't take a value. Should we follow the same pattern here? i.e. presence of tpu-use-quic parameter indicates the use of quic..